### PR TITLE
Refine UG Installation, Add, Help

### PR DESCRIPTION
**Pull request by lipwei1808**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ Please read through sections [Installation](#installation) and [Command Breakdow
  
    <img src="images/Ui.png" width="650">
 
-5. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will
-   open the help window.
+5. Type a command in the command box and press `Enter` to execute it. e.g. typing `help` and pressing `Enter` will
+   execute the `help` command and open the help window.
 
-   To get started with UniCa$h, you can run the `add_transactions` command!
+   To get started with UniCa$h, you can run the (`add_transactions` command)[#add-transaction]!
```
Comment (woojiahao) [inline docs/UserGuide.md:42]: Screenshot 2023-11-13 at 16 42 53

This link is not working, believe you flipped the order of parentheses: i.e. `[]()`

**Review by woojiahao — CHANGES_REQUESTED** [review]
LGTM apart from the broken link

File: docs/UserGuide.md (hunk 1 above)
Comment (lipwei1808) [reply docs/UserGuide.md:42]: 🤦‍♂️

**Review by woojiahao — APPROVED** [review]
LGTM

**Pull request merged**